### ISSUE TITLE
Error: spawn nix-hash ENOENT

### ISSUE THREAD

**Issue by jluttine** [rvl/bower2nix#16]
I tried using `bower2nix` as follows in a directory containing `bower.json`:
```
nix-shell --pure -p nodePackages.bower2nix
bower2nix > bower-generated.nix
```

The content of `bower-generated.nix`:
```
# Generated by bower2nix v3.2.0 (https://github.com/rvl/bower2nix)
{ fetchbower, buildEnv }:
buildEnv { name = "bower-env"; ignoreCollisions = true; paths = [
  # failed to fetch "angular": nix-hash failed to exec with error Error: spawn nix-hash ENOENT
  # failed to fetch "angular-bootstrap": nix-hash failed to exec with error Error: spawn nix-hash ENOENT
  # failed to fetch "jquery-timepicker": nix-hash failed to exec with error Error: spawn nix-hash ENOENT
  # failed to fetch "jstzdetect": nix-hash failed to exec with error Error: spawn nix-hash ENOENT
  # failed to fetch "ical.js": nix-hash failed to exec with error Error: spawn nix-hash ENOENT
  # failed to fetch "fullcalendar": nix-hash failed to exec with error Error: spawn nix-hash ENOENT
  # failed to fetch "hsl_rgb_converter": nix-hash failed to exec with error Error: spawn nix-hash ENOENT
  # failed to fetch "angular-mocks": nix-hash failed to exec with error Error: spawn nix-hash ENOENT
]; }
```

Any ideas what's wrong?

**Comment by jluttine** [comment]
Oh, needed to add `nix` package to nix-shell. So, perhaps it's a packaging issue on nixpkgs side that this was missing.

**Comment by rvl** [comment]
Thanks -- I think bower2nix should be changed to have `nix-hash` in its path.

**Comment by rvl** [comment]
Nixpkgs PR merged NixOS/nixpkgs#30978
